benchmarking/nursery/odsc_tutorial/transformer_wikitext2/sagemaker/plot_results.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Dict, Any, Optional
+
+from syne_tune.experiments import ComparativeResults, PlotParameters, SubplotParameters
+from benchmarking.nursery.odsc_tutorial.transformer_wikitext2.benchmark_definitions import (
+    benchmark_definitions,
+)
+
+
+TMLR10_SETUPS = [
```

Comment:
Nit: what is TMLR10 and how does it relate to ODSC?

Reply:
I am using results here I did for another purpose. They are real expensive. And I wanted the scripts to work for me.